swagger.yaml

```
@@ -6655,6 +6655,11 @@ definitions:
       file_name:
         type: string
         description: The file name prefaced by the folder that contains it.
+      is_custom_manifest:
+        description: Is this a custom (user uploaded) manifest?
```

I think we should explain to users what it means when this is `false`, as they don't know the service itself might be also generating manifests for their cluster

```suggestion
        description: Indicates whether this is a manifest that was manually uploaded by the user, as opposed to being internally generated by the Assisted Service
```